### ISSUE TITLE
Email continuity / ActionMailbox with Postal server

### ISSUE THREAD

**Issue by jtraulle** [chatwoot/chatwoot#1959]
**Is your enhancement request related to a problem? Please describe.**
Per GDPR requirements, I prefer when possible not involve other data processors (like Sendgrid, Mailgun, etc.) and thus I have deployed my own mailserver using Postal.

**Describe the solution you'd like**
It will be great if conversation continuity could work with a postal server but for now, despite my numerous trials, I could not succeed to get it to work with Postal.

I have tried with different endpoints (postmark, relay, etc.) without luck (with and without password.

I also put you in the Additional context section a link to a blog article that seems promising ans screenshots of Postal web-ui.

**Additional context**

- [Postal, the open source mail server (official website)](https://postal.atech.media)
- [Postal, the open source mail server (Github repository)](https://github.com/postalhq/postal)

- [[HowTo] Rails ActionMailbox with Postal (blog article)](https://lcx.wien/blog/rails-actionmailbox-with-postal/)

![postal-http-endpoint](https://user-images.githubusercontent.com/613615/111903033-7cb97100-8a40-11eb-941b-bfc080ac7420.png)
_Postal HTTP Endpoint configuration_

![postal-routing-routes](https://user-images.githubusercontent.com/613615/111903038-804cf800-8a40-11eb-84fb-bbd68f7d9c8b.png)
_Postal incomming Routing configuration_

![postal-routes-detail](https://user-images.githubusercontent.com/613615/111903040-83e07f00-8a40-11eb-99eb-7fee54c3f912.png)
_Postal route detail_

**Comment by pranavrajs** [comment]
@jtraulle I guess this is more of a documentation problem, rather than an issue with the app. Shall I move this to the repo?

**Comment by jtraulle** [comment]
@pranavrajs Yes, feel free to move it to the other repository if you think this is more a documentation problem ;)

**Comment by sojan-official** [comment]
ref:  https://lcx.wien/blog/rails-actionmailbox-with-postal/ 

we would need to add an ingress controller to support postal

**Comment by jtraulle** [comment]
Hey :)

Anything I can do to help on this ?

**Comment by sojan-official** [comment]
@jtraulle Thanks for the offer. If the community is willing to raise a PR based on https://lcx.wien/blog/rails-actionmailbox-with-postal/ , we can work with them on getting it merged. 
Otherwise, I will reach out once we have some bandwidth to pick this up.

**Comment by nlkocx** [comment]
+1 for Postal Server. I don't want to use PostMark. I have my own MailServer.

**Comment by lenikadali** [comment]
Hello @sojan-official 

I would like to pick up this issue as well. Just to confirm, the only thing needed here is an implementation based on https://lcx.wien/blog/rails-actionmailbox-with-postal/ or does this require documentation as well as some coding?

**Comment by sojan-official** [comment]
> Hello @sojan-official
> 
> I would like to pick up this issue as well. Just to confirm, the only thing needed here is an implementation based on https://lcx.wien/blog/rails-actionmailbox-with-postal/ or does this require documentation as well as some coding?



> Hello @sojan-official
> 
> I would like to pick up this issue as well. Just to confirm, the only thing needed here is an implementation based on https://lcx.wien/blog/rails-actionmailbox-with-postal/ or does this require documentation as well as some coding?

We will also need the documentation !!! but that can be take up as a separate task. Thanks a lot for looking into this. !
